indigo/distribution.yaml

```diff
@@ -4580,6 +4580,45 @@ repositories:
       url: https://github.com/GT-RAIL/jaco_gazebo.git
       version: develop
     status: maintained
+  jaguar:
+    doc:
+      type: git
+      url: https://github.com/gstavrinos/jaguar.git
+      version: kinetic-devel
+    release:
+      packages:
+      - jaguar_control
+      - jaguar_description
+      - jaguar_msgs
+      - jaguar_navigation
+      tags:
+        release: release/kinetic/{package}/{version}
```

Comment:
You can't just copy-n-paste for each distribution. You need to run `bloom` for each of them separately since it generates specific files for each supported platform.